ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/eos_cli_config_gen.schema.yml

```diff
@@ -2417,6 +2417,14 @@ keys:
                       min: 5000
                       max: 3600000
                       description: Template interval in milliseconds
+              table_size:
+                type: int
+                convert_types:
+                - str
+                min: 1
```

Comment:
if we know the maximum value, please set it.

Reply:
Maximum depends on the platform. I have not found documentation of a definite upper limit, but on the platforms I tested, the minimum was always 1, hence why I defined a minimum but no max.